libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -771,6 +763,8 @@ void RCOutput::trigger_groups(void)
 
 void RCOutput::timer_tick(void)
 {
+    safety_update();
```

Comment:
The comment above this function that states timer_tick is only needed for oneshot mode is no longer true. (It also appears the comment is stale as this also is responsible for doing dshot updates)